wiki/Beatmap_Spotlights/id.md

```diff
@@ -0,0 +1,202 @@
+   - Setiap beatmap yang dikurasi, dipilih dan dikunci sebelum musim dimulai.
+   - Setelah musim dimulai, seluruh jadwal akan mulai dirilis.
+3. Setiap musim dibagi menjadi beberapa minggu. Setiap minggu diberi label dengan surat.
+   - Setiap huruf mewakili playlist multiplayer lobby mingguan.
+   - Putaran mingguan dilaksanakan sepanjang musim dan diulangi 2 kali.
+4. Setelah satu musim berakhir, akan terdapat jeda 3 minggu sebelum musim baru dimulai. Selama jeda waktu ini, Penyesuaian terhadap proyek dapat dilakukan.
+
+### Leaderboard Musiman
+
+*Fitur ini masih menunggu untuk di implementasi. Sebagian akan ditambahkan seiring berjalannya musim. Maka dari itu, hal ini dapat berubah sewaktu-waktu.*
+
+Leaderboard musiman ini merangkum hasil skor ranking mingguan dari setiap peserta. Berdasar leaderboard musiman, setiap peserta akan ditempatkan ke kelompok liga yang mencerminkan penempatan relatif mereka dalam Leaderboard.
+
+1. Leaderboard musiman adalah jumlah yang ditimbang dari keseluruhan Leaderboard mingguan yang dicapai di Timeshift Lobbies.
+2. Peserta hanya dapat memiliki satu skor peringkat mingguan per Playlist.
+   - Memutar ulang sebuah playlist pada minggu yang lain hanya akan memperbarui skor terbaik dan skor baru yang lebih baik ini akan menimpa skor lama yang lebih buruk.
+3. Peserta tidak akan diberi tahu tentang peringkat musiman mereka. Sebagai gantinya, mereka akan dimasukan kedalam braket liga.
+   - Hanya peringkat 50 teratas di Leaderboard musiman yang peringkat tepatnya akan terlihat.
+   - Braket ditetapkan berdasarkan peringkat dalam Leaderboard. Peserta akan dikelompokan kedalam braket yang sesuai dengan peringkat mereka.
+4. Braket yang ditetapkan akan diumumkan setelah minggu ke dua pada 1 musim.
+
+### Hadiah
+
+*Sebagian dari hadiah ini masih menunggu implementasi. Mereka akan ditambahkan seiring berjalannya musim. Maka dari itu, hal ini dapat berubah sewaktu-waktu.*
+
+Hadiah akan dibagikan kepada timeshift lobby mingguan, mapper dari beatmap yang dikurasi, dan setiap peserta yang mendapat peringkat di Leaderboard musiman.
+
+1. 10 teratas dari setiap timeshift lobby mingguan akan diberikan hadiah osu! supporter tag selama 1 minggu.
+2. Selama musim berlangsung, setiap peserta akan ditempatkan ke braket, yang ditunjukkan oleh Lencana Profil Unik yang terdapat di profil mereka selama musim berlangsung.
+   - Lencana akan diperbarui setiap minggu setelah minggu ke dua pada 1 musim.
+   - Pemain terbaik selama 1 musim mungkin saja dapat mempertahankan lencana tersebut secara permanen. Namun, rincian tentang hal tersebut masih belum diputuskan.
+3. Di setiap akhir musim, mapper terbaik, sesuai keputusan curator, akan mendapatkan hadiah supporter tag selama 1 bulan.
+4. Di setiap akhir musim, medali baru yang tidak bisa dibuka akan ditambahkan. Pemain harus menyelesaikan setiap beatmap musim ini setidaknya satu kali untuk mendapatkannya.
+
+| Lencana | Bracket Tier | Peringkat |
```

Comment:
`Bracket Tier` should be translated into something like `Tingkat Pencapaian` i suppose? i'm not really sure but it definitely shouldn't be left untranslated

Reply:
agak ragu kmrn pas mau translate ini wkwk oke udah diganti ke Tingkat Pencapaian